scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -18,11 +18,27 @@ task GetToolVersions {
   String version = "unspecified"
 
   String effective_version = select_first([git_branch_or_tag, version])
+
+  String workspace_id_output = "workspace_id.txt"
+  String workspace_name_output = "workspace_name.txt"
+  String workspace_namespace_output = "workspace_namespace.txt"
+  String workspace_bucket_output = "workspace_bucket.txt"
+  String submission_id_output = "submission_id.txt"
+  String workflow_id_output = "workflow_id.txt"
+  String google_project_output = "google_project.txt"
+
   command <<<
     # Prepend date, time and pwd to xtrace log entries.
     PS4='\D{+%F %T} \w $ '
     set -o errexit -o nounset -o pipefail -o xtrace
 
+    # Scrape out various workflow / workspace info from the localization and delocalization scripts.
+    sed -n -E 's!.*gs://fc-(secure-)?([^\/]+).*!\2!p' /cromwell_root/gcs_delocalization.sh | sort -u > ~{workspace_id_output}
+    sed -n -E 's!.*gs://(fc-(secure-)?[^\/]+).*!\1!p' /cromwell_root/gcs_delocalization.sh | sort -u > ~{workspace_bucket_output}
+    sed -n -E 's!.*gs://fc-(secure-)?([^\/]+)/submissions/([^\/]+).*!\3!p' /cromwell_root/gcs_delocalization.sh | sort -u > ~{submission_id_output}
+    sed -n -E 's!.*gs://fc-(secure-)?([^\/]+)/submissions/([^\/]+)/([^\/]+)/([^\/]+).*!\5!p' /cromwell_root/gcs_delocalization.sh | sort -u > ~{workflow_id_output}
+    sed -n -E 's!.*(terra-[0-9a-f]+).*# project to use if requester pays$!\1!p' /cromwell_root/gcs_localization.sh | sort -u > ~{google_project_output}
+
```

Comment:
are we planning to merge this code? It would be great to have this added to Utils